pypeit/edgetrace.py

```diff
@@ -1186,6 +1083,9 @@ def save(self, outfile=None, overwrite=True, checksum=True, float_dtype='float32
         #   - PCA type, used for rebuilding the PCA when loading
         prihdr['PCATYPE'] = ('None' if self.pca is None else self.pca_type,
                              'PypeIt: Edge trace PCA type')
+        #   - The dispersion name is needed to setup some spectrographs
+        if self.spectrograph.dispname is not None:
```

Comment:
This is somewhat convoluted, so I'm open to a better solution if you have one: `dispname` is needed because it's used by some spectrographs to setup `spec_min_max`.  See, e.g., `spectrographs.gemini_gnirs.spec_min_max`. This is my solution to the issue you were having with needing to add `spec_min` and `spec_max` to `EdgeTraceSet`.  I prefer this approach, but I admit the connection between requiring `dispname` so that you can specify `spec_min_max` isn't intuitive.

Reply:
I am not sure I follow or like this solution but admit that mine was not any better. My suggestion is that we remove dispname here as I don't see why this code needs it. It is not using this information at all. 

The slittrace is the only object that needs specmin and specmax and you are grabbing this information out of the spectrograph object without any reference to the disperse.

Reply:
If I remember right, I need the disperser when the edgetrace file is read back in.  That's why it's being read from the master file header.  E.g., when you run `pypeit_chk_edges` you need this bit of code.